dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.tsx

```diff
@@ -89,16 +96,27 @@ export const ConfigPanel = ({ visualizations, setCurVisId, updateIsDataConfigOpt
     []
   );
 
-  // To check , If user empty any of the value options
-  const isInvalidValueOptionConfig = useMemo(() => !(vizConfigs.dataConfig && (vizConfigs.dataConfig?.valueOptions?.xaxis?.length !== 0
-    && vizConfigs.dataConfig?.valueOptions?.yaxis?.length !== 0)), [vizConfigs])
+  // To check, If user empty any of the value options
+  const isValidValueOptionConfigSelected = useMemo(() => {
+    const valueOptions = vizConfigs.dataConfig?.valueOptions;
+    const { Bar, Line, TimeSeries, Histogram, Pie, TreeMap, Gauge, HeatMap } = visChartTypes;
+    return valueOptions && (
```

Comment:
Can we create variables to store multiple conditions to make this code more readable?

Reply:
Done, Thanks Amruta!